mv/core/src/main/scala/org/apache/carbondata/mv/rewrite/Utils.scala

```diff
@@ -590,6 +591,21 @@ object Utils extends PredicateHelper {
     }
   }
 
+  /**
+   * transform cast expression to change it's child attribute reference name to lower case
+   */
+  def getTransformedCastExpression(cast: Cast): Expression = {
+    cast.transform {
+      case attr: AttributeReference =>
+        CarbonToSparkAdapter.createAttributeReference(attr.name.toLowerCase,
```

Comment:
move `attr.name.toLowerCase` to next line

Reply:
done